testing/kokoro/system_tests.sh

```diff
@@ -137,7 +137,7 @@ fi
 
 
 
-if [[ $KOKORO_BUILD_ARTIFACTS_SUBDIR = *"system-tests"* && -n $GOLANG_SAMPLES_GO_VET ]]; then
+if [[ $KOKORO_BUILD_ARTIFACTS_SUBDIR = *"system-tests"* && -n $GOLANG_SAMPLES_GO_VET ]] || [[ $CHANGED_DIRS =~ "run" ]]; then
```

Comment:
Please also check `GOLANG_SAMPLES_GO_VET` -- that way it only runs on one version of Go.

Reply:
done.